py/tests/test_api_ext.py

```diff
@@ -23,12 +23,21 @@
 from swagger_client.apis.external_api import ExternalApi
 from swagger_client.api_client import ApiClient
 from swagger_client.models.block import Block
+from swagger_client.models.signed_tx import SignedTx
+from swagger_client.models.coinbase_tx import CoinbaseTx
 
 def utc_now():
     d = datetime.datetime.utcnow()
     epoch = datetime.datetime(1970,1,1)
     return int((d - epoch).total_seconds())
 
+def signed_coinbase_tx(height):
+    account = "BAAggMEhrC3ODBqlYeQ6dk00F87AKMkV6kkyhgfJ/luOzGUC+4APxFkVgAYPai3TjSyLRObv0GeDACg1ZxwnfHY="
+    coinbase = CoinbaseTx(pubkey = account, nonce = height - 1)
+    return SignedTx(data = coinbase, type = "coinbase",
+            signatures =
+            ["MEUCIQDRusihjYzeL1VgYhruK3v9M9ktwmr5TGmaor6kTPpA5gIgehFypmZyXn7VZNxHhiq/FN4Us7vsVM2frKZeQpLsxPE="])
```

Comment:
How did you compute this? If it comes from computation (i.e. it is not random string), you may replace this hardcoded string with the related python computation.

Reply:
Actually this is the product of
https://github.com/aeternity/epoch/blob/master/apps/aecore/src/aec_keys.erl#L152
using a private key, corresponding to the pub_key in the same function, signing the coinbase transaction. It should be a valid signature.